PluginsAndFeatures/azure-toolkit-for-eclipse/com.microsoft.azuretools.core/src/com/microsoft/azuretools/core/ui/WindowsAzurePreferencePage.java

```diff
@@ -133,18 +134,20 @@ public boolean performOk() {
 					String oldPrefVal = DataOperations.getProperty(dataFile, Messages.prefVal);
 					DataOperations.updatePropertyValue(doc, Messages.prefVal,
 							String.valueOf(btnPreference.getSelection()));
-					String version = DataOperations.getProperty(dataFile, Messages.version);
+					
+					final String version = DataOperations.getProperty(dataFile, Messages.version);
+					final String newVersion = Activator.getDefault().getBundle().getVersion().toString(); 
```

Comment:
Not sure of this. Extract the original version to a variable only. Let me try to understand the code.